internal/integrations/github/github_test.go

```diff
@@ -0,0 +1,78 @@
+package github
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/google/go-github/v26/github"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockChecksService struct {
+	mock.Mock
+}
+
+func (m *MockChecksService) ListCheckRunsForRef(context context.Context, owner string, repo string, ref string, opt *github.ListCheckRunsOptions) (*github.ListCheckRunsResults, *github.Response, error) {
+	args := m.Called()
+	results := args.Get(0)
+
+	if results != nil {
+		return args.Get(0).(*github.ListCheckRunsResults), nil, nil
+	}
+
+	return nil, nil, args.Error(2)
+
+}
+
+func TestGetTravisCIBuildURLForRef_Success(t *testing.T) {
+	ctx := context.Background()
+	g := New(ctx, "Wattpad", "fake-access-token")
+
+	m := new(MockChecksService)
+	m.On("ListCheckRunsForRef").Return(&github.ListCheckRunsResults{
+		CheckRuns: []*github.CheckRun{
+			&github.CheckRun{
```

Comment:
Doesn't make much difference, but this `&github.CheckRun` syntax can be omitted when defining non-composite-literal types (slices and maps). Since the type of an element of a `[]*github.CheckRun` is implied, I like removing the extraneous syntax.